test/LibraryManager.IntegrationTest/FilesDeployer.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    /// <summary>
+    /// Deploys assembly manifest resources.
+    /// </summary>
+    public static class FilesDeployer
```

Comment:
If you use a `[DeploymentItem]` on the test assets folder, you don't need to do the whole embedded resource thing.  We do that internally to solve deployment issues that this project should never have.

Reply:
Will use this